README.md

```diff
@@ -55,6 +55,18 @@ If `cc` is not on your `PATH` as an alias to your C compiler, you may need to sp
 CC=gcc make
 ```
 
+## How to test darkhttpd
```

Comment:
Please move this to the end of the file (like L187) because most users shouldn't have to run tests.

Reply:
That makes sense. Done!